### DISCUSSION TITLE
blog/ai-tools-collection/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#329] [Announcements]
# blog/ai-tools-collection/

本篇博客旨在为大家推荐几款实用且易于上手的 AI 工具，希望能够帮助大家提高工作效率，更好地应用 AI 技术。

https://jimmysong.io/blog/ai-tools-collection/

**Comment by blazefengyi** [comment]
感谢分享，也分享一个AI工具站，目前收录了国内外的 2660+ AI工具，还不错

**Reply by blazefengyi** [reply]
忘了网址了，哈哈，51GPT，https://www.51gpt.net/

**Comment by Deyong888** [comment]
感谢分享，我也分享几款目前用得比较顺的网站：
1. AI未来城--AI工具导航站
网址：https:// www.linktoai.top
用户痛点： 面对众多的AI工具，如何快速找到最好用最合适的工具成为了一大难题。
解决方案： AI未来城是一个专注于AI工具导航的平台，汇集了目前市场上各种热门的AI工具，持续收录排名较前的顶级AI工具，每款AI只挑选出最顶级的TOP10工具。无论你需要文本生成、图像处理还是音乐创作，这里都有详尽的分类和推荐，帮助你快速找到所需的工具，提高创作效率。

2. 星界AI--一站式AIGC创作平台
网址： https://www.chatallais.com
用户痛点： 在不同平台之间切换，容易浪费时间和精力，无法专注于创作本身。
解决方案： 星界AI提供了一站式的AIGC创作解决方案，涵盖AI生成文本、AI生成图片、AI生成音乐、AI情感聊天、AI智能问答等多种功能。用户可以在一个平台上完成所有创作任务，节省时间，提升创作体验。


16. Soundraw
网址： https://www.soundraw.com

用户痛点： 需要高质量的音乐素材，但没有时间或精力进行原创音乐制作。

解决方案： Soundraw 是一个 AI 音乐生成平台，提供多种类型的音乐风格和曲调，帮助你快速找到合适的背景音乐，提升作品的整体效果。

17. Art AI
网址： https://artiv.com

用户痛点：  想尝试不同的艺术风格，但缺乏绘画技巧。

解决方案： Art AI 是一款基于 AI 的艺术风格转换工具，可以将你的照片或图片转换成各种艺术风格，如梵高、毕加索、莫奈等，满足你探索艺术的兴趣。

NightCafe Creator
网址： https://creator.nightcafe.studio
用户痛点：  需要独特且高质量的 AI 艺术作品，但缺乏艺术创作经验。
解决方案： NightCafe Creator 是一个 AI 艺术生成平台，提供多种 AI 艺术风格和创作工具，帮助你轻松生成独特的艺术作品，并提供社区分享和交流功能。

Pictory.ai
网址： https://www.pictory.ai
用户痛点：  需要将长视频内容转成短视频，但缺乏专业的视频剪辑技能。
解决方案： Pictory.ai 是一款 AI 视频剪辑工具，可以将长视频内容自动剪辑成短视频，并添加字幕、音乐等，帮助你快速制作短视频内容。

**Comment by rootsongjc** [comment]
这里面有些工具现在已经不推荐使用了，比如 Notion AI、Smodin，基本可以被 ChatGPT 代替；或者在中国大陆无法使用，比如 Bing Chat（现在叫 Copilot）。
